ring/src/lib.rs

```diff
@@ -20,6 +20,12 @@ pub mod ring_verifier;
 
 pub type RingProof<F, CS> = Proof<F, CS, RingCommitments<F, <CS as PCS<F>>::C>, RingEvaluations<F>>;
 
+// Crates that are required by the downstream dependencies.
+pub mod prelude {
+    pub use fflonk;
+    pub use merlin;
+}
```

Comment:
I think most crypto crates reexport from the root, not from a prelude module.

A prelude module is when you need to glob import many things together.   I guess that's what you want though, so `use ring_proof::prelude::*;` gives both?

Reply:
You're right. I'll fix it